package.json

```diff
@@ -11,7 +11,7 @@
     "start": "node ./dist/rpx-exui/api/server.bundle.js",
     "start:local": "cd api && yarn start",
     "build:node": "cd api && yarn build",
-    "build:prod": "ng build --prod",
+    "build:prod": "node --max_old_space_size=8192 node_modules/@angular/cli/bin/ng build --prod",
```

Comment:
@ronaldmansveld - is the `node_modules/@angular/cli/bin` bit necessary, or can it just be `node --max_old_space_size=8192 ng build --prod`?